### PR TITLE
Simplified Cloud AI API integration.

### PR DESCRIPTION
**Pull request by kornelregius**
Only commits on Jan 11, 2021 have file changes.

### CODE REVIEW

**Comment by kornelregius** [comment]
Waiting for master branch success build.

File: dev.Dockerfile [hunk 1]
```diff
@@ -13,10 +13,12 @@
 #   # you can run the tests against this new image using:
 #   ./test -i kaggle/python-dev -p test_user_secrets.py
 #
-FROM gcr.io/kaggle-images/python:staging
 
+#FROM gcr.io/kaggle-images/python:staging
```
Comment (Philmod) [inline dev.Dockerfile:17]: Did you mean to comment this line?

**Review by Philmod — COMMENTED** [review]
The build seems broken, for an unrelated problem `ERROR: Cannot uninstall 'llvmlite'` (b/177279594)

File: dev.Dockerfile (hunk 1 above)
Comment (kornelregius) [reply dev.Dockerfile:17]: Sorry, it is a merge issue. Changed back.

**Review by rosbo — APPROVED** [review]
Once #933 is merged, you can pull the latest changes from `master` and the build should be green.

**Comment by kornelregius** [comment]
I think it ready to merge
https://ci.kaggle.net/blue/organizations/jenkins/kernels%2Fdocker-python/detail/PR-932/4/pipeline

**Pull request merged**